smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/KafkaConnector.java

```diff
@@ -84,7 +84,7 @@
 @ConnectorAttribute(name = "value.serializer", type = "string", direction = Direction.OUTGOING, description = "The serializer classname used to serialize the payload", mandatory = true)
 @ConnectorAttribute(name = "acks", type = "string", direction = Direction.OUTGOING, description = "The number of acknowledgments the producer requires the leader to have received before considering a request complete. This controls the durability of records that are sent. Accepted values are: 0, 1, all", defaultValue = "1")
 @ConnectorAttribute(name = "buffer.memory", type = "long", direction = Direction.OUTGOING, description = "The total bytes of memory the producer can use to buffer records waiting to be sent to the server.", defaultValue = "33554432")
-@ConnectorAttribute(name = "retries", type = "long", direction = Direction.OUTGOING, description = "Setting a value greater than zero will cause the client to resend any record whose send fails with a potentially transient error.", defaultValue = "2147483647")
+@ConnectorAttribute(name = "retries", type = "long", direction = Direction.OUTGOING, description = "A value equals to the default one will cause the client to try to resend any record whose send fails with a potentially transient error during an amount of time configured by `delivery.timeout.ms`. Otherwise, setting a value greater than zero will cause the client to resend any record whose send fails with a potentially transient error until the number of retries is reached.", defaultValue = "2147483647")
```

Comment:
```suggestion
@ConnectorAttribute(name = "retries", type = "long", direction = Direction.OUTGOING, description = "If set to a positive number, the connector will try to resend any record that was not delivered successfully (with a potentially transient error) until the number of retries is reached. If set to 0, retries are disabled. If not set, the connector tries to resend any record that failed to be delivered (because of a potentially transient error) during an amount of time configured by `delivery.timeout.ms`.", defaultValue = "2147483647")
```